docs/swagger.yaml

```diff
@@ -88,6 +86,11 @@ paths:
                       turnoverShare: about 25%
                       prepurchaseTime: 15 days
                       averageRentalDuration: 3 weeks
+                    prepaidServices:
+                      turnoverShare: about 25%
+                      prepurchaseTime: 10 weeks
```

Comment:
Like the re-visor comment: Would it help us (down the line, maybe) if we used number of days? Maybe `prepurchaseDays: 70 days`?